utils/docker/Dockerfile.ubuntu.18.04

```diff
@@ -28,7 +28,8 @@ RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y \
             libssl-dev libtool-bin libyaml-dev                          \
             locales make meson nasm ninja-build pandoc patch            \
             pylint python-dev python3-dev scons sg3-utils uuid-dev      \
-            yasm valgrind libhwloc-dev man python-distro
+            yasm valgrind libhwloc-dev man python-distro openjdk-8-jdk  \
+	    maven
```

Comment:
This needs to be upgraded to 20.04, i am afraid now.

Reply:
done